doc/classes/RDVertexAttribute.xml

```diff
@@ -10,14 +10,19 @@
 	</tutorials>
 	<members>
 		<member name="format" type="int" setter="set_format" getter="get_format" enum="RenderingDevice.DataFormat" default="218">
+			The way this attribute's data is interpreted when sent to a shader.
 		</member>
 		<member name="frequency" type="int" setter="set_frequency" getter="get_frequency" enum="RenderingDevice.VertexFrequency" default="0">
+			The rate at which this attribute is pulled from it's vertex buffer.
 		</member>
 		<member name="location" type="int" setter="set_location" getter="get_location" default="0">
+			The location in the shader that this attribute is bound to.
```

Comment:
This property seems to be missing an unit of measure, if it needs one.

Reply:
No, its more like a unique ID for each attribute than a specific location in memory.